src/main/webapp/i18n/en/statistics.json

```diff
@@ -0,0 +1,37 @@
+{
+    "statistics": {
+        "amountOfStudents": "# of students",
+        "span": {
+            "day": "Day",
+            "week": "Week",
+            "month": "Month",
+            "year": "Year"
+        },
+        "title": "User statistics",
+        "days": " days",
+        "loggedInUsers": "Amount of logins of distinct users in the last ",
+        "activeUsers": "Amount of logins of active users in the last ",
+        "totalSubmissions": "Total amount of submissions in the last ",
+        "exercises": {
+            "released": "Total amount of released exercises in the last ",
+            "due": "Total amount of exercise deadlines in the last "
+        },
+        "conductedExams": "Total amount of conducted exams in the last ",
+        "examParticipation": " -> amount of student who participated: ",
+        "examRegistrations": " -> with the amount of students being registered in these exam: ",
+        "activeTutors": "Total amount of tutors which have created an assessment in the last ",
+        "results": "Total amount of results created in the last ",
+        "resultFeedbacks": "-> feedback count: ",
+        "userLogins": "User Logins",
+        "submissions": "Submissions"
+    },
+    "weekdays": {
```

Comment:
get rid of this

Reply:
[Done](https://github.com/ls1intum/Artemis/pull/2471/commits/499bb9fd70d8f4710fe3744189d0493f61a8db7d)